climpred/metrics.py

```diff
@@ -155,17 +122,138 @@ def get_metric_function(metric):
                 'msss'
                 'nmae'
                 'uacc'
+                'msss_murphy'
+                'bias'
+                'bias_slope'
+                'conditional_bias'
+                'std_ratio'
+                'crps'
+                'crpss'
+                'less'
                 """)
         return metric
 
 
-# TODO: Do we need wrappers or should we rather create wrappers for skill score
-#       as used in a specific paper: def Seferian2018(ds, control):
-#       return PM_compute(ds, control, metric=_ppp, comparison=_m2e)
-def _ppp(ds, control, comparison, running=None, reference_period=None):
+# wrap xskillscore metrics to work with comparison argument
+def _pearson_r(forecast, reference, dim='svd', comparison=None):
+    return pearson_r(forecast, reference, dim=dim)
+
+
+def _pearson_r_p_value(forecast, reference, dim='svd', comparison=None):
+    return pearson_r_p_value(forecast, reference, dim=dim)
+
+
+def _mse(forecast, reference, dim='svd', comparison=None):
+    return mse(forecast, reference, dim=dim)
+
+
+def _rmse(forecast, reference, dim='svd', comparison=None):
+    return rmse(forecast, reference, dim=dim)
+
+
+def _mae(forecast, reference, dim='svd', comparison=None):
+    return mae(forecast, reference, dim=dim)
+
+
+def _crps(forecast, reference, dim='svd', comparison=None):
+    return crps_ensemble(forecast, reference).mean(dim)
+
+
+def _crps_gaussian(forecast, mu, sig, dim='svd', comparison=None):
+    return crps_gaussian(forecast, mu, sig).mean(dim)
+
+
+def _crpss(forecast, reference, dim='svd', comparison=None):
+    """
+    Continuous Ranked Probability Skill Score.
+    Reference
+    ---------
+    * Matheson, James E., and Robert L. Winkler. “Scoring Rules for Continuous
+      Probability Distributions.” Management Science 22, no. 10 (June 1, 1976):
+      1087–96. https://doi.org/10/cwwt4g.
+    Range
+    -----
+    perfect: 0
+    max: 0
+    else: negative
+    """
+    mu = reference.mean(dim)
+    sig = reference.std(dim)
+    ref_skill = _crps_gaussian(forecast, mu, sig, dim=dim)
+    forecast_skill = _crps(forecast, reference, dim=dim)
+    skill_score = (ref_skill - forecast_skill) / ref_skill
+    return skill_score
+
+
+def _less(forecast, reference, dim='svd', comparison=None):
+    """
+    Logarithmic Ensemble Spread Score
+    Reference
+    ---------
+    * Kadow, Christopher, Sebastian Illing, Oliver Kunst, Henning W. Rust,
+      Holger Pohlmann, Wolfgang A. Müller, and Ulrich Cubasch. “Evaluation of
+      Forecasts by Accuracy and Spread in the MiKlip Decadal Climate Prediction
+      System.” Meteorologische Zeitschrift, December 21, 2016, 631–43.
+      https://doi.org/10/f9jrhw.
+    Range
+    -----
+    pos: under-disperive
+    neg: over-disperive
+    perfect: 0
+    """
+    numerator = _mse(forecast, reference, dim=dim)
+    try:  # PM
+        denominator = reference.std('lead').mean(dim)
+    except:  # hindcast
```

Comment:
I think linting will not like a plain except statement without a specific error it's trying to catch like:
`except TypeError`?

Reply:
I got rid of those now.